Source/WebInspectorUI/UserInterface/Views/ConsoleGroup.js

```diff
@@ -34,10 +34,12 @@ WI.ConsoleGroup = class ConsoleGroup extends WI.Object
         super();
 
         this._parentGroup = parentGroup;
+        this._element = false;
     }
 
     // Public
-
```

Comment:
Style: please keep this newline

Reply:
Addressed in e2c3874b5b35844fbcd028d68867b71662ae988e.